auth0/src/main/java/com/auth0/android/provider/CallbackHelper.java

```diff
@@ -47,9 +48,19 @@ public CallbackHelper(@NonNull String packageName) {
      * @return the callback URI.
      */
     public String getCallbackURI(@NonNull String domain) {
-        String uri = String.format(REDIRECT_URI_FORMAT, domain, packageName);
-        Log.v(TAG, "The Callback URI is: " + uri);
-        return uri;
+        HttpUrl url = HttpUrl.parse(domain);
+        if (url == null) {
```

Comment:
The error message will be generated by the webpage. I don't know what's better, either crash the app when he presses the Social Button or log the error but keep the app running.